src/bundle/Resources/translations/fieldview.en.xlf

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file date="2017-06-15T17:29:53Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="977cb55c24b1708d46ccbd753c2cea745976ae21" resname="fieldview.field.empty">
+        <source>fieldview.field.empty</source>
+        <target state="new">fieldview.field.empty</target>
```

Comment:
Unless I'm mistaken, the translation system will load the translations from PlatformUI, since the domain is the same. hybrid-platform-ui defines its own sources, that's all (since it has those as source string as well).

Or am I missing something ?